app/controllers/fact_categories_controller.rb

```diff
@@ -0,0 +1,51 @@
+class FactCategoriesController < ApplicationController
+  before_action :authenticate, :set_variables, :check_authorization
+
+  # TODO(Ozzie): Should the api be .../fact_categories or .../categories ?
+  # POST /organizations/:organization_id/fact_categories
+  # POST /organizations/:organization_id/fact_categories.json
+  def create
+    if @fact_category.save
+      render json: FactCategoryRepresenter.new(@fact_category).to_json
+    else
+      render json: @fact_category.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /organizations/:organization_id/fact_categories/:id
```

Comment:
I was just mimicing what we do in other controllers, but I can remove the .json line.